recipes/scikit-bio/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "0.4.2" %}
+
+package:
+  name: scikit-bio
+  version: {{ version }}
+
+source:
+  fn: scikit-bio-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-bio/scikit-bio-{{ version }}.tar.gz
+  md5: bbbe704410c59083e9eb65fa9d01a50d
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bz2file >=0.98
+    - lockfile >=0.10.2
+    - cachecontrol >=0.11.5
+    - contextlib2 >=0.4.0
+    - decorator >=3.4.2
+    - future >=0.14.3
+    - ipython >=3.2.0
+    - matplotlib >=1.4.3
+    - natsort >=4.0.3
+    - numpy >=1.9.2
+    - pandas >=0.17.0, <=0.17.1
```

Comment:
For some reason I can't find `RangedIndex` in `master`. :confused: Though I don't use `pandas` really. So I probably have a lot to learn. We could try running with `0.17.1` and see if it works. Though it will probably be a few hours before we get all the binaries generated.